### ISSUE TITLE
Setup SMTP server

### ISSUE THREAD

**Issue by lmullane** [bcgov/foi-flow#139]
Title of ticket: 

#### Description
Summarize issue 

#### Dependencies
Are there any dependencies?

#### DOD
- [ ] List the items that need to be complete for this ticket to be considered done
- [ ] 
- [ ] 
- [ ] 
- [ ]

**Comment by lmullane** [comment]
Hey @abin-aot, I've checked out the [NR Common Email Service](https://digital.gov.bc.ca/common-components/common-hosted-email-service) and they handle sending emails from your app but not receiving emails back into your app.

It got me thinking about if our users need to be able to reply to an email and have it be captured in the app, as opposed to receive the email notification and need to login to the app before they can respond or take action.

Here's a couple of underpinning assumptions:

- IAO staff and ministry coordinators will be logged into the application all day. In app notifications will be more useful and they will action the request in the app. They may even want the option of turning off email notifications?
- Emails can be forwarded or intercepted. How do I know that the person responding to an email is in fact the user I am trying to reach? For security, what if the app sends an email notification to an applicant that their package is ready for download, but requires that they login securely to the application to access their package. This will ensure the package does not end up with the wrong recipient.

Are there use cases where we see the need for someone to be able to respond by email and have it captured by the app?

I believe we are thinking of the something like the respond to this email feature in GitHub. While it is nice to have, I wonder if we really need it.


cc @mpilchar @m-prodan @arielleandrews

**Comment by abin-aot** [comment]
Here are my understanding on this ,as per the UX we have now.

- Correspondence section on UX, which has TO & FRO communication between the applicants and IAO  analysts will work only if the applicant has provided email to us on the request

- The concern which I have is NOT with sending emails from our app, but with receiving emails on to our APP via  IAO's generic /group email address. It seems I need more information on this area and functionality etc.

- Because of the above concern, I am curious to know "what we meant with Correspondence Logs"


Please let me know if we need to catch up on this

**Comment by lmullane** [comment]
@abin-aot, agreed there is no concern with sending emails. Question is do we need to receive emails in the app? I'm not convinced it's necessary.

Let's discuss tomorrow at standup.

**Comment by mpilchar** [comment]
Discussion on June 29/21 with product team determined that inbound email would be doable and desirable for applicant. Changes DOD - new task created

**Comment by mpilchar** [comment]
I've confirmed with the team who built CHES that best way to reach out with any questions is via Rocketchat https://chat.developer.gov.bc.ca/channel/nr-common-services-showcase
